include/zephyr/bluetooth/mesh.h

```diff
@@ -48,6 +48,8 @@
 #include <zephyr/bluetooth/mesh/od_priv_proxy_cli.h>
 #include <zephyr/bluetooth/mesh/sol_pdu_rpl_srv.h>
 #include <zephyr/bluetooth/mesh/sol_pdu_rpl_cli.h>
+#include <zephyr/bluetooth/mesh/brg_cfg_cli.h>
```

Comment:
Could you synchronize prefixes in public API and headers with @omkar3141? Seems Omkar uses `sbr` (that seems more reasonable to me). I guess whole subnetwork bridge functionality within Zephyr mesh should have common naming rules.

Reply:
I agree that we should sync. I have taken the especially from the Mesh Protocol spec, as I found that most sensible. It is easier to look up the spec that way.